test/config.js

```diff
@@ -22,6 +21,13 @@ if (!window.__karma__) {
     callback = window.__karma__.start;
 }
 
+// Add qunit-fixture to page if not present
```

Comment:
Why remove this from the html and add it this way? I feel like the html solution is a little simpler.